.github/workflows/ci.yml

```diff
@@ -0,0 +1,101 @@
+        runs-on: ubuntu-latest
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    -   php: '5.5'
+                        composer-flags: '--prefer-lowest'
+                    -   php: '5.5'
+                        symfony: '^3.0'
+                    -   php: '5.6'
+                    -   php: '7.0'
+                    -   php: '7.1'
+                        symfony: '^4.0'
+                    -   php: '7.2'
+                        symfony: '^5.0'
+                    -   php: '7.3'
+                    -   php: '7.4'
+                    -   php: '8.0'
+                        #symfony: '^6.0' #TODO: allow it in separate PR
+                        calculate-code-coverage: 'yes'
+
+        steps:
+            -   name: Checkout source
+                uses: actions/checkout@v2
+
+            -   name: Get code coverage driver
+                uses: actions/github-script@v3.1
+                id: code-coverage-driver
+                with:
+                    script: 'return "${{ matrix.calculate-code-coverage }}" == "yes" ? "pcov" : "none"'
+                    result-encoding: string
+
+            -   name: Get symfony flex version
+                uses: actions/github-script@v3.1
+                id: flex-version
+                with:
+                    script: 'return "${{ matrix.php }}" == "7.0" ? "~1.6.0" : "^1.11"'
+                    result-encoding: string
+
+            -   name: PHP setup
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: ${{ steps.code-coverage-driver.outputs.result }}
+
+            -   name: Downgrade composer for olf Symfony Flex
+                if: steps.flex-version.outputs.result == '~1.6.0'
+                run: composer self-update --1
+
+            -   name: Install Symfony Flex
+                if: startsWith(matrix.php, '5') == false
+                run: composer global require -o --no-interaction symfony/flex:${{ steps.flex-version.outputs.result }}
+
+            -   name: Require Symfony version
+                if: matrix.symfony
+                run: composer config extra.symfony.require ${{ matrix.symfony }}
+
+            -   name: Lower minimum stability to test with Symfony 6
+                if: matrix.symfony == '^6.0'
+                run: composer config minimum-stability beta
+
+            -   name: Update composer dependencies
+                run: composer update -o --no-interaction --no-progress ${{ matrix.composer-flags }}
+
+            -   name: Run test suite
+                run: ./vendor/bin/phpunit -v
+
+            -   name: Upload coverage results to Coveralls
+                if: matrix.calculate-code-coverage == 'yes'
+                env:
+                    COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
ref https://github.com/php-coveralls/php-coveralls/pull/325#discussion_r748888064

let's still try to see if works
we cannot change GITHUB_TOKEN secret anymore

Reply:
done - and changed main commend to close #326 issue